test/dataverseClient.spec.ts

```diff
@@ -1481,4 +1483,32 @@ describe('DataverseClient', () => {
       })
     })
   })
-})
\ No newline at end of file
+
+  describe('updateDataset()', () => {
+    it('should call axios with expected url', async () => {
+      const datasetId = random.number().toString()
+      const datasetInformation: BasicDatasetInformation = {
+        title: 's',
+        descriptions: [{ text: 'some ', date: '2019-09-09' }],
+        authors: [
+          {
+            fullname: 'tester tests'
+          }
+        ],
+        contact: [{ email: 'tai@theagilemonkeys.com', fullname: 'Tai Nguyen' }],
```

Comment:
random.internet package also lets you generate email addresses and names I believe 🚀

Reply:
Done.